### PR TITLE
Add button (and more)

### PR DESCRIPTION
**Pull request by riraum**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,15 +1,33 @@
...
-  // gridSquares.style.width = "20px";
-  // gridSquares.style.height = "20px";
-  // gridSquares.style.background = "red";
+const container = document.querySelector("#container");
+const divSquares = document.querySelectorAll("div-squares");
+// const gridSquares = document.createElement("div-squares");
+
+function createGrid(gridAmount) {
+  for (i = 0; i < gridAmount; i++) {
+    let gridSquares = document.createElement("div-squares");
```
Comment (nobe4) [inline script.js]: `"div-square"` is incorrect in this context.
It should be `"div"` only.

See https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement

File: script.js [hunk 2]
```diff
@@ -1,15 +1,33 @@
...
+    container.appendChild(gridSquares);
+  }
+}
+createGrid(256);
+
+function createUserGrid(gridAmount) {
+  // container.removeChild(container.divSquares);
+  container.replaceChildren();
+  for (i = 0; i < gridAmount; i++) {
+    let gridSquares = document.createElement("div-squares");
```
Comment (nobe4) [inline script.js]: Same

File: style.css [hunk 3]
```diff
@@ -1,15 +1,20 @@
...
   border: 1px solid black;
-  min-width: 50px;
-  min-height: 50px;
+  min-width: 15px;
+  min-height: 15px;
   background-color: red;
+  /* flex: 1; */
 }
 
 div-squares:hover {
```
Comment (nobe4) [inline style.css]: this is incorrect css, you need to add `.` before the class names

File: script.js [hunk 4]
```diff
@@ -1,15 +1,35 @@
...
-  // gridSquares.style.background = "red";
+const container = document.querySelector("#container");
+const divSquares = document.querySelectorAll("div");
+// const gridSquares = document.createElement("div-squares");
+
+function createGrid(gridAmount) {
+  for (i = 0; i < gridAmount; i++) {
+    let gridSquares = document.createElement("div");
+    container.appendChild(gridSquares);
+    gridSquares.classList.add("squares");
```
Comment (nobe4) [inline script.js]: Add the class first
```suggestion
    gridSquares.classList.add("squares");
    container.appendChild(gridSquares);
```

File: script.js [hunk 5]
```diff
@@ -1,15 +1,35 @@
...
+}
+createGrid(256);
+
+function createUserGrid(gridAmount) {
+  // container.removeChild(container.divSquares);
+  container.replaceChildren();
+  for (i = 0; i < gridAmount; i++) {
+    let gridSquares = document.createElement("div");
+    container.appendChild(gridSquares);
+    gridSquares.classList.add("squares");
```
Comment (nobe4) [inline script.js]: this might be refactored into a function

Comment (riraum) [reply script.js]: I was considering that. Finally did it and added the loop to the function too, thank you!

**Pull request merged**